supervisord.conf

```diff
@@ -1,6 +1,6 @@
 [program:smarts]
 environment=PYTHONUNBUFFERED=1
-command=python examples/single_agent.py scenarios/intersections/6lane
+command=python examples/single_agent.py scenarios/intersections/2lane
```

Comment:
why was this change made?  (I'm not necessarily against it, I'm just curious.)

Reply:
I was trying to reproduce the bug. And the bug involved using 2lane intersection. So i had to change the config file. The workstation I got doesn't seem to be working so i have to push changes to a branch and pull them on remote to test them out there.

Reply:
no problem, although I think we should avoid pushing unrelated changes.  we should back this one out now that you've tested your code.